Dev ali oss persistor (#6266)
RasaHQ/rasa: docs/docs/user-guide/cloud-storage.mdx

  If there is no container with the name `AZURE_CONTAINER`, Rasa will create it.

* Other Remote Storage

>   Other remote storage is supported using the external model persistor library,
>   which you can install or implement by yourself.

Reviewer: ```suggestion
  Support for other remote storages can be achieved by using an external library or implementing
  a custom module for it by extending the `rasa.nlu.persistor.Persistor` class.
```

>   Start the Rasa server with ``remote-storage`` option set to the module path of model persistor.

Reviewer: I suggest to give an example of how the command would look like

Author: I had add a single command

  If there is no container with the name `AZURE_CONTAINER`, Rasa will create it.

* Other Remote Storage

  Support for other remote storages can be achieved by using an external library or implementing
  a custom module for it by extending the `rasa.nlu.persistor.Persistor` class.

>   Start the Rasa server with ``remote-storage`` option set to the module path of model persistor.

Reviewer: ```suggestion
  Start the Rasa server with `remote-storage` option set to the module path of model persistor.
```

>   A single command is:

Reviewer: ```suggestion
  An example command is:
```

  ```bash

>   rasa run --remote-storage rasa.nlu.persistor.Persistor

Reviewer: That should be a custom module. People might misunderstand if we give the parent class as example here.
```suggestion
  rasa run --remote-storage <your module>.<class name>
```
